.asf.yaml

```diff
@@ -31,3 +31,15 @@ github:
     squash:  true
     merge:   false
     rebase:  false
+  protected_branches:
+    master:
+      required_status_checks:
+        strict: true
+        contexts:
+          - CI
+          - Single
+          - Storage
+          - Cluster
```

Comment:
These contexts are not exactly the same as what we see in the PR page, thus I fork and set in my forked repo and use GitHub API to see what the `context` is, hopefully it's right

![image](https://user-images.githubusercontent.com/15965696/98217309-4abddb80-1f85-11eb-9351-0a4480d0aa90.png)
![image](https://user-images.githubusercontent.com/15965696/98217329-51e4e980-1f85-11eb-8638-f3394e60ab5c.png)